fe/fe-core/src/main/java/org/apache/doris/persist/RefreshExternalTableInfo.java

```diff
@@ -0,0 +1,102 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.persist;
+
+import com.clearspring.analytics.util.Lists;
+import org.apache.doris.catalog.Column;
+import org.apache.doris.common.io.Writable;
+import org.apache.doris.common.io.Text;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+import java.util.List;
+
+public class RefreshExternalTableInfo implements Writable {
+    public static final Logger LOG = LoggerFactory.getLogger(RefreshExternalTableInfo.class);
+
+    private String dbName;
```

Comment:
Use GSON

Reply:
Sorry, Firstly I try use GSON to do serialization. But failed when I need to deserialize the `newSchema`. It seems `Column` cause the problem.